Nothing uses this.

From 6cd8ac42e8c296eb1c9e154b3e99dab8bffcad0a Mon Sep 17 00:00:00 2001
From: Steve Lhomme <robux4@ycbcr.xyz>
Date: Thu, 12 Apr 2018 13:24:38 +0200
Subject: [PATCH] configure: remove minizip check not referenced anywhere

(cherry picked from commit d22de99eb8c46786ca2feb4abca2e4f25ceeb0a8)
--- a/configure.ac
+++ b/configure.ac
@@ -890,23 +890,14 @@ AS_IF([test -n "${PKG_CONFIG}" ],[
 
 
 dnl
-dnl Check for zlib.h and -lz along with system -lminizip if available
+dnl Check for zlib.h and -lz if available
 dnl
 AC_CHECK_HEADERS(zlib.h, [ have_zlib=yes ], [ have_zlib=no ])
 AM_CONDITIONAL(HAVE_ZLIB, [ test "${have_zlib}" = "yes" ])
 if test "${have_zlib}" = "yes"
 then
   VLC_ADD_LIBS([sap],[-lz])
-  PKG_CHECK_MODULES([MINIZIP], [minizip] , [ have_minizip=yes ], [
-    AC_CHECK_HEADERS([unzip.h], [
-      have_minizip=yes
-      MINIZIP_LIBS="-lminizip -lz"
-    ], [
-      have_minizip=no
-    ])
-  ])
 fi
-AM_CONDITIONAL(HAVE_MINIZIP, [ test "${have_minizip}" = "yes" ])
 
 
 dnl
-- 
2.41.0

